lib/rucio/daemons/hermes/hermes.py

```diff
@@ -76,267 +67,13 @@ def default(datetype: Union[datetime.date, datetime.datetime]) -> str:
-        except stomp.exception.NotConnectedException as error:
-            logger(
-                logging.WARNING,
-                "[broker] Could not deliver message due to NotConnectedException: %s",
-                str(error),
-            )
-            continue
-        except stomp.exception.ConnectFailedException as error:
-            logger(
-                logging.WARNING,
-                "[broker] Could not deliver message due to ConnectFailedException: %s",
-                str(error),
-            )
-            continue
-        except Exception as error:
-            logger(logging.ERROR, "[broker] Could not deliver message: %s", str(error))
-            continue
-
-        if str(message["event_type"]).lower().startswith("transfer") or str(
-            message["event_type"]
-        ).lower().startswith("stagein"):
-            logger(
-                logging.DEBUG,
-                "[broker] - event_type: %s, scope: %s, name: %s, rse: %s, request-id: %s, transfer-id: %s, created_at: %s",
-                str(message["event_type"]).lower(),
-                message["payload"].get("scope", None),
-                message["payload"].get("name", None),
-                message["payload"].get("dst-rse", None),
-                message["payload"].get("request-id", None),
-                message["payload"].get("transfer-id", None),
-                str(message["created_at"]),
-            )
-
-        elif str(message["event_type"]).lower().startswith("dataset"):
-            logger(
-                logging.DEBUG,
-                "[broker] - event_type: %s, scope: %s, name: %s, rse: %s, rule-id: %s, created_at: %s)",
-                str(message["event_type"]).lower(),
-                message["payload"].get("scope", None),
-                message["payload"].get("name", None),
-                message["payload"].get("rse", None),
-                message["payload"].get("rule_id", None),
-                str(message["created_at"]),
-            )
-
-        elif str(message["event_type"]).lower().startswith("deletion"):
-            if "url" not in message["payload"]:
-                message["payload"]["url"] = "unknown"
-            logger(
-                logging.DEBUG,
-                "[broker] - event_type: %s, scope: %s, name: %s, rse: %s, url: %s, created_at: %s)",
-                str(message["event_type"]).lower(),
-                message["payload"].get("scope", None),
-                message["payload"].get("name", None),
-                message["payload"].get("rse", None),
-                message["payload"].get("url", None),
-                str(message["created_at"]),
-            )
-        else:
-            logger(logging.DEBUG, "[broker] Other message: %s", message)
-    return to_delete
-
-
-def deliver_emails(
-        messages: "Iterable[dict[str, Any]]",
-        logger: "LoggerFunction"
-) -> list[str]:
+def deliver_emails(messages: "Iterable[dict[str, Any]]", logger: "LoggerFunction") -> list:
```

Comment:
The return type here was changed to a generic `list`. Did you mean to change it to `list[int]` to be consistent with `deliver_messages`?